pkg/webboot/webboot.go

```diff
@@ -0,0 +1,9 @@
+package webboot
+
+//Distro struct defines an operating system distrubution
```

Comment:
you can remove the word struct from this comment. Maybe someday it gets changed to be an interface, and the comment would still be accurate as long as the word struct is not there.
distribution, not distrubution

Reply:
Done.